packages/ui/components/TasksTable/Projectdetails.tsx

```diff
@@ -102,14 +107,14 @@ export default function ProjectDetails({ project }) {
         <>
           {project.tasks.map((task) => (
             <TaskRow
-              key={task.taskId}
+              key={JSON.stringify(task)}
```

Comment:
Why do we need to do this? If `task` changes due to an update, the key will change, which we likely don't want.

Reply:
@brettstack program details (which includes tasks) is stored in react query context, and to avoid refetching the entire program with every task update, I'm patching the program details using `queryClient.setData` function. So to force taskRow component rendering I need to provide different keys everytime the task details got updated.

This video (captured by @Ramya) demonstrate the issue that I solved with adding `JSON.stringify` to the keys 
https://drive.google.com/file/d/1-vZen8J4MhRWhbvBjMHiwciEgGwb22oV/view